modules/apps/asset/asset-tags-admin-web/src/main/java/com/liferay/asset/tags/admin/web/internal/portlet/AssetTagsAdminPortlet.java

```diff
@@ -101,6 +102,9 @@ public void editTag(
 			// Update tag
 
 			_assetTagService.updateTag(null, tagId, name, serviceContext);
+
+			CTCollectionTimelineUtil.setCTTimelineKeys(
```

Comment:
This is a processAction.  Isn't it too late to set it here?  at this point the user is saving to the database.